include/deal.II/integrators/l2.h

```diff
@@ -146,6 +146,10 @@ namespace LocalIntegrators
      *
      * \f[ \int_Z fv\,dx \quad \text{or} \quad \int_F fv\,ds \f]
      *
+     * The vector <tt>input</tt> is $f$ evaluated at the quadrature points in
+     * the finite element. Therefore, its size must be equal to the number of
+     * quadrature points in the element.
```

Comment:
Would you mind (here and below) documenting the other parameters as well and use `@p ...` instead of `<tt>...</tt>`? In particular, "factor" does not yet appear anywhere in the documentation.

Reply:
Added proper param documentation for all in `LocalIntegrators::L2`